firmware/library/L1_Drivers/system_clock.hpp

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <cstdint>
+
+class SystemClock
+{
+ public:    
+      virtual void selectClockSource(bool option) = 0;
+      virtual void setMainPLL() = 0;
```

Comment:
What does this exactly do? Does this set the main PLL or does it set this clock to the main PLL?